src/index.html

```diff
@@ -2,16 +2,162 @@
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="birth_date">
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="agreement">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldest">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            autocomplete="off"
+            minlength="5"
+            maxlength="32"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldest">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="brands"
+            multiple
+            >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldest">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea name="comments">
+          </textarea>
```

Comment:
Now this textarea has a content(spaces)  `          `
```suggestion
          <textareaname="comments"></textarea>
```